nominatim_geolocalize/README.rst

```diff
@@ -0,0 +1,23 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+
+======================
+Nominatim Geolocalize
+======================
+
+Get partner coordinate using Nominatim API
+
+Alternative to Google API
+
+
+
+Credits
+=======
+
+Contributors ------------
```

Comment:
You can use readme by fragments for having this auto-populated. See template module: https://github.com/OCA/maintainer-tools/tree/master/template/module

Reply:
Thanks @pedrobaeza, I renamed the module as per your suggestion, regarding the README, this is already done, only I kept README.rst for better readability of PR, but I understand this should be replaced by auto-populated one, no ?

Reply:
Yes, sorry for not checking that part.